src/components/qcResultsUpload/QcResultsUploadForm.vue

```diff
@@ -71,18 +72,27 @@ export default {
       usedBySelected: 'extraction',
       busy: null,
       disableUpload: null,
+      uploadSuccessful: null,
     }
   },
   computed: {
     qcResultUploadsRequest: ({ api }) => api.traction.qc_results_uploads.create,
     border() {
-      if (this.file === null) return 'border-0'
-      else {
+      if (this.uploadSuccessful === true) {
```

Comment:
May be It would be better to have a comment here to avoid ambiguity in future that  describes the expected styles associated with three states - true, false and null